package.json

```diff
@@ -9,7 +9,8 @@
     "path-posix": "^1.0.0",
     "phosphor-disposable": "^1.0.5",
     "phosphor-signaling": "^1.2.0",
-    "requirejs": "^2.2.0"
+    "requirejs": "^2.2.0",
+    "url": "^0.11.0"
```

Comment:
I'm actually not using `url`, it should have been `url-join`, since `url` didn't offer anything over `posix-path`.  Agreed that we don't want to have our own path implementation.